resources/views/layouts/app.blade.php

```diff
@@ -16,7 +16,11 @@
   <!-- Scripts -->
   <!-- Font Awesome -->
   <script src="https://use.fontawesome.com/5fd8ad5172.js"></script>
-
+  <script type="text/javascript">
+  $( "#dropdown" ).select2({
```

Comment:
This does not belong here, move it to the app.js file with the rest of the .select2 declares